pkg/util/fs_util.go

```diff
@@ -57,7 +57,7 @@ func ExtractFileSystemFromImage(img string) error {
 
 // PathInWhitelist returns true if the path is whitelisted
 func PathInWhitelist(path, directory string) bool {
-	if path == constants.KanikoExecutor {
+	if path == constants.KanikoExecutor || path == constants.KanikoCerts {
```

Comment:
Do we want to just have a list of "privileged paths" in constants, in case we ever have to add any more?

Reply:
for sure, just added it!